docs/SITEMAP-GUIDE.md

```
@@ -0,0 +1,166 @@
+   # Verify sitemap exists before copying
+   if [ -f "dist/sitemap.xml" ]; then
+       cp dist/sitemap.xml /home/user/public_html/sitemap.xml
+   else
+       echo "Error: dist/sitemap.xml not found"
+       exit 1
+   fi
+   ```
+
+2. **In cPanel → Cron Jobs**, add:
+
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)